test_programs/compile_failure/unconstrained_slice/src/main.nr

```diff
@@ -0,0 +1,11 @@
+unconstrained fn uncon_slice() -> [Field] {
```

Comment:
We already have a `brillig_slice_to_acir` test

Reply:
I have removed this test.